ow_dynamic_terrain/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ow_dynamic_terrain)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-# add_compile_options(-std=c++11)
-add_compile_options(-std=c++11)
```

Comment:
Just curious: is the removal of this add_compile_options and the others a requirement for Noetic, or an unrelated cleanup?

Reply:
I am not sure why this was added in first place (probably the default template by catkin adds that) it might have been required at some point. Both **melodic** and **noetic** are currently embracing c++14 standard so having this module requesting c++11 resulted in a set of compile time errors for **noetic** (Ubuntu 20.04 has an upgraded version of the c/c++ compiler and libraries).